### ISSUE TITLE
Accessibility information added per component

### ISSUE THREAD

**Issue by IrlaRebelo** [SAP/fundamental-styles#1430]
Hello,
>>Feature requests must include:

Talking with Deno this week, we agreed that it would be beneficial to add a section "Accessibility" on the right side list per component.

despite the fact that many components would have little or special conditions to fulfill Accessibility, any information available to users would help them understand what to expect and what is necessary to do to deliver accessibility to that component. A simple note "this component should be used in addition with other resources in order to fulfill accessibility requirement ACC XXX" is a enough to inform that the team fundamental cares about this topic.

I put myself available to discuss the 31 SAP requirements (specially those that are mandatory for Designers).

Best regards,
Irla Rebelo
